pinot-spi/src/main/java/org/apache/pinot/spi/utils/JsonUtils.java

```diff
@@ -200,6 +201,11 @@ public static JsonNode bytesToJsonNode(byte[] jsonBytes)
     return DEFAULT_READER.readTree(new ByteArrayInputStream(jsonBytes));
   }
 
+  public static Map<String, Object> jsonNodeToObject(JsonNode jsonNode)
```

Comment:
Rename this to `jsonNodeToMap`, and remove the one using the `MAPPER` which has worse performance IIRC

Reply:
`jsonNodeToMap` already exists. Removing this method and changing its usages.